### DISCUSSION TITLE
Pros and Cons of Components Libaries

### DISCUSSION THREAD

**Discussion by starkris51** [bcc-code/bcc-design#303] [Ideas]
Prime Vue
Pros:
- Good Examples
- Has the most components

Cons:
- Code Typos in Documentation
- Experienced Issues with setup even if following documentation
- Unintuitive styling with tailwind

Radix Vue
Pros: 
- Easy to style with tailwind
- Good documentation

Cons: 
- Many missing components

**Comment by laurensgroeneveld** [comment]
Haven't looked into it deeply, but I'm impressed with the Radix implementation in #301 as opposed to the manual pagination component of #300 - styling it seems really easy and the functionality seems very similar.

Lack of components is a good thing to look at - if they mostly have components that we already have, there's not a lot of benefit in switching over.

With PrimeVue I would also be a bit concerned about how easy to style it is, as it seems more opinionated, and one strong reason for the design library is that we implement a BCC-specific style the designers create.

I'm not sure if a decision has been reached yet if we want to move away from implementing components ourselves (if not, I'll happily be involved to think about it, just add me to a meeting), because a third option might be to look at that: what are the pain points, can we make it easier to implement new components, or is it not really feasible to build an extensive library with the limited manpower that we have?
